kroxylicious-sample/src/test/java/io/kroxylicious/sample/SampleFilterIT.java

```diff
@@ -39,7 +39,7 @@
 import static org.junit.jupiter.api.Assertions.fail;
 
 @ExtendWith(KafkaClusterExtension.class)
-class SampleFilterIntegrationTest {
+class SampleFilterIT {
```

Comment:
Adopts the standard naming expected by the failsafe plugin.  No point fighting the convention.

Reply:
I agree there is no point in fighting the convention but the convention makes me sad